### PR TITLE
SDK-2134 Add example for identity profiles in IDV

### PR DESCRIPTION
**Pull request by dhivya-pattabhiraman**
Added example for Identity profile checks using IDV sessions.

### CODE REVIEW

File: examples/doc-scan-identity-checks/src/controllers/index.controller.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+  const sessionSpec = new SessionSpecificationBuilder()
+    .withClientSessionTokenTtl(600)
+    .withResourcesTtl(90000)
+    .withUserTrackingId('some-user-tracking-id')
+    .withSubject(subject)
+    .withIdentityProfileRequirements(identityProfileRequirements)
+    .withSdkConfig(
+      new SdkConfigBuilder()
+        .withAllowsCameraAndUpload()
```
Comment (laurent-yoti) [inline examples/doc-scan-identity-checks/src/controllers/index.controller.js:38]: Should we remove that or is it mandatory (I know the API wants it, but I can not remember if it needs to be specified)?

Comment (dhivya-pattabhiraman) [reply examples/doc-scan-identity-checks/src/controllers/index.controller.js:38]: Actually, lemme check with Pawel and find out what needs to be specified and what not, and accordingly add/remove them. I am also wondering about the mobile handoff.

Comment (dhivya-pattabhiraman) [reply examples/doc-scan-identity-checks/src/controllers/index.controller.js]: Updated the example configuration to:

```  const sessionSpec = new SessionSpecificationBuilder()
    .withClientSessionTokenTtl(600)
    .withResourcesTtl(90000)
    .withUserTrackingId('some-user-tracking-id')
    .withSubject(subject)
    .withIdentityProfileRequirements(identityProfileRequirements)
    .withSdkConfig(
      new SdkConfigBuilder()
        .withPrimaryColour('#2d9fff')
        .withLocale('en-GB')
        .withPresetIssuingCountry('GBR')
        .withSuccessUrl(`${config.YOTI_APP_BASE_URL}/success`)
        .withErrorUrl(`${config.YOTI_APP_BASE_URL}/error`)
        .build()
    )
    .build();```

Comment (dhivya-pattabhiraman) [reply examples/doc-scan-identity-checks/src/controllers/index.controller.js]: Tested it in prod with this config and I was able to create a session successfully.

**Review by laurent-yoti — APPROVED** [review]

**Review by laurent-yoti — APPROVED** [review]

**Pull request merged**